slug.go

```diff
@@ -183,6 +190,19 @@ func (p *Packer) packWalkFn(root, src, dst string, tarW *tar.Writer, meta *Meta,
 			return err
 		}
 
+		if !filepath.IsAbs(src) {
+			src, err = filepath.Abs(src)
+			if err != nil {
+				return fmt.Errorf("failed to get absolute path for src %q: %w", src, err)
+			}
+		}
```

Comment:
Since src is a parameter of packWalkFn, does this need to be performed on every WalkFunc call?